inference-engine/src/mkldnn_plugin/nodes/mkldnn_eltwise_node.cpp

```diff
@@ -1611,23 +1626,21 @@ void MKLDNNEltwiseNode::executeReference() {
 }
 
 void MKLDNNEltwiseNode::execute(mkldnn::stream strm) {
-    std::vector<const uint8_t *> src_ptrs(inputNum);
-    for (int i = 0; i < inputNum; i++)
-        args_ptrs.src_ptr[i] = reinterpret_cast<const uint8_t*>(memPtrs[i]->GetData()) + start_offset_in[i];
-    args_ptrs.dst_ptr = reinterpret_cast<uint8_t*>(memPtrs.back()->GetData()) + start_offset_out;
-
-    // In general case we need to recompute offsets as well but currently all supported layout assumes batch to be outermost dimension
-    if (isDynBatchEnabled)
-        dims_out[batchDimIdx] = static_cast<size_t>(batchToProcess());
-
-    if (eltwise_kernel) {
-        if (tensorRank == optimalTensorRank) {
-            executeOptimized6D();
-        } else {
-            executeOptimizedGeneric();
+    if (pPrim) {
+        jit_eltwise_call_args_ptrs args_ptrs = {};
+        VectorDims dims_out = pPrim->getJep().dims;
```

Comment:
` it seems to me that this parameter should be recalculated on inferShapes() or prepareParams() stages`

yes this parameter recalculated on prepareParams() stage, but after cache introduction we no need to recalculate `dims_out` and we can take them from `jep`

`Moreover, all this dynBatch stuff should work via the dynamic shape.`
I still not sure how we will be process dynamic batch cases.
Because if we will be process it as dynamic case we got performance drop at least at first.
So let's discuss it offline

Reply:
make `dims_out` as static array